### ISSUE TITLE
Stopping: 'install' failed for 'HTML::Tidy'.

### ISSUE THREAD

**Issue by devharsh** [petdance/html-tidy#23]
```
OS: Windows 10 Pro
Distribution: Strawberry Perl

Note:
Alien::Tidyp is up to date (v1.4.7).

Error:

Running make test
"C:\Strawberry\perl\bin\perl.exe" -MExtUtils::Command::MM -e cp_nonempty -- Tidy.bs blib\arch\auto\HTML\Tidy\Tidy.bs 644
"C:\Strawberry\perl\bin\perl.exe" "-MExtUtils::Command::MM" "-MTest::Harness" "-e" "undef _Test::Harness::Switches; test_harness(0, 'blib\lib', 'blib\arch')" t/_.t
t/00-load.t .......... # Testing HTML::Tidy 1.56, Perl 5.024000; tidyp 1.04
t/00-load.t .......... ok
t/cfg-for-parse.t .... ok
t/clean-crash.t ...... ok
t/extra-quote.t ...... ok
t/ignore-text.t ...... ok
t/ignore.t ........... ok
t/levels.t ........... ok
t/message.t .......... ok
t/opt-00.t ........... ok
t/perfect.t .......... ok
t/pod-coverage.t ..... skipped: Test::Pod::Coverage 1.04 required for testing POD coverage
t/pod.t .............. ok
t/roundtrip.t ........ ok
t/segfault-form.t .... ok
t/simple.t ........... ok
t/too-many-titles.t .. ok
t/unicode-nbsp.t .....
# Failed test 'Perl chars OK'

t/unicode-nbsp.t ..... 1/2 #   at t/unicode-nbsp.t line 19.
# got: '&nbsp;
# '
# expected: '&nbsp;
# '
# Failed test 'Byte string OK'
# at t/unicode-nbsp.t line 20.
# got: '&nbsp;
# '
# expected: '&nbsp;
# '
# Looks like you failed 2 tests of 2.

t/unicode-nbsp.t ..... Dubious, test returned 2 (wstat 512, 0x200)
Failed 2/2 subtests
t/unicode.t .......... ok
t/venus.t ............ ok
t/version.t .......... ok
t/wordwrap.t ......... ok
## Test Summary Report

t/unicode-nbsp.t   (Wstat: 512 Tests: 2 Failed: 2)
  Failed tests:  1-2
  Non-zero exit status: 2
Files=21, Tests=68,  3 wallclock secs ( 0.19 usr +  0.05 sys =  0.23 CPU)
Result: FAIL
Failed 1/21 test programs. 2/68 subtests failed.
dmake.exe:  Error code 255, while making 'test_dynamic'
  PETDANCE/HTML-Tidy-1.56.tar.gz
  C:\STRAWB~1\c\bin\dmake.exe test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:
  reports PETDANCE/HTML-Tidy-1.56.tar.gz
Stopping: 'install' failed for 'HTML::Tidy'.
```

**Comment by LeeThompson** [comment]
Same result on Strawberry Perl 5.8.14 (Windows 7 x64)

**Comment by soonix** [comment]
This appears to be the same as issue #11 - lines ending with CRLF instead of LF alone.

**Comment by petdance** [comment]
Fixed with f4ff587
